src/containers/Sidebar/NavbarBrand.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { useSelector } from 'react-redux';
 
 import { Link } from 'react-router-dom';
 
-const NavbarBrand = ({ title, brand_image, isSidebarCollapsed }) => {
-	const href = brand_image.href ?? "/";
+const NavbarBrand = ( props ) => {
+	const theme = useSelector(state => state.theme);
+	const title = useSelector(state => state.config.title);
+	const isSidebarCollapsed = useSelector(state => state.sidebar.isSidebarCollapsed);
```

Comment:
@petrKavulok Here I would be more careful

	const isSidebarCollapsed = useSelector(state => state.sidebar?.isSidebarCollapsed);